routes.yaml

```diff
@@ -0,0 +1,6 @@
+app_name: dissolution-api
+group: api
+weight: 900
+routes:
+  1: ^/companies/(.*)/close-a-company/.*
```

Comment:
Ok I don't think we need the companies prefix, as per the API Definition document, can we make this:
```
^/dissolution-request/.*
```
which will require you to update https://github.com/companieshouse/dissolution-api/blob/master/src/main/java/uk/gov/companieshouse/controller/HealthcheckController.java#L14 from "dissolution" -> "dissolution-request".

@szmc FYI this may require the Karate test to be updated.